packages/vxrn/src/exports/prebuild.ts

```diff
@@ -111,18 +111,9 @@ export const prebuild = async ({
         'react,react-native,expo',
       ])
 
-      try {
-        const packageJsonPath = path.join(root, 'package.json')
-        let packageJsonContents = await FSExtra.readFile(packageJsonPath, 'utf8')
-
-        packageJsonContents = packageJsonContents.replace(/expo run:ios/g, 'one run:ios')
-        packageJsonContents = packageJsonContents.replace(/expo run:android/g, 'one run:android')
```

Comment:
Is there a reason for removing this automatic replacement? I think we can still try to do this automatically, and if failed, show a warning with manual instructions. 

As there are several duplicated issues opened before by people not noticing this, and the expo commands will start Metro automatically, making weird errors while trying to run RN.

Reply:
I hate when clis modify files which i didn't ask them to modify, so i think we shouldn't worsen the situation and we should file an issue on the expo repo.

Reply:
plus automatic replacement is bad, it can replace things that were intended

Reply:
I agree, but I think we need to do so since expo will modify that file now anyway, and it's unsure if they will accept the change to not do so.

Meanwhile, rather than it being automatically modified and broken, it's better to at least let it be not broken imo.

Reply:
We can try to let Expo not modify package.json (tried before and seems there's no easy way so I end up with this quick solution), maybe backup the file and restore it after Expo has done the work. Or use a better method to apply that patch.

Reply:
Oh I didn’t know expo modified the file? I thought we were just doing that ourselves. If they are modifying it and we are just modifying it back, it’s funny, but yea I think that’s worth us doing then.

Reply:
Yea we are just kind of modifying it back since expo is doing so lol.